src/FINDOLOGIC/Export/Data/Image.php

```diff
@@ -40,7 +41,7 @@ class Image implements Serializable
 
     public function __construct($url, $type = self::TYPE_DEFAULT, $usergroup = '')
     {
-        $this->url = $url;
+        $this->url = DataHelper::emptyValueCheck($url);
```

Comment:
[Suppress](https://phpmd.org/documentation/suppress-warnings.html) the Code Climate warning for using static access here, because this is a legitimate use case.

Reply:
Done